navigation.html

```diff
@@ -0,0 +1,148 @@
+  style="width: 885px; height: 582px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin" class="img-nav">
+</iframe>
+         
+<p class="navtext-text">Contains the sub-nav categories under main nav tabs. Useful for bigger Applications and empowers a good UX.</p>
+    <div class="nav-component">
+        
+
+<div class="navbar-sample">
+    <a href="index.html">Home</a>
+    <div class="subnav-sample">
+      <button class="subnavbtn">About <i class="fa fa-caret-down"></i></button>
+      <div class="subnav-content">
+        <a href="#company">Company</a>
+        <a href="#team">Team</a>
+        <a href="#careers">Careers</a>
+      </div>
+    </div> 
+    <div class="subnav-sample">
+      <button class="subnavbtn">Services <i class="fa fa-caret-down"></i></button>
+      <div class="subnav-content">
+        <a href="#bring">Ecommerce</a>
+        <a href="#deliver">Video Player</a>
+        <a href="#package">Social Media</a>
+        <a href="#express">Quiz Game</a>
+      </div>
+    </div> 
+    <div class="subnav-sample">
+      <button class="subnavbtn">Skills <i class="fa fa-caret-down"></i></button>
```

Comment:
on hover, this button background color is not covering the entire width of the button.